### PR TITLE
Enable Segoe Variable

### LINKED ISSUE #12452: Rejuvenated UI doesn’t pick up Segoe UI Variable on Windows 11
What it says on the tin.

### PR DESCRIPTION
**Pull request by zadjii-msft**
By adding another entry to our `maxversiontested`s.

Screenshots in https://github.com/microsoft/terminal/issues/12452#issuecomment-1035356054

* [x] Closes #12452 
* [x] I work here
* [x] Docs are fine
* [x] Tests are fine

### CODE REVIEW

File: src/cascadia/WindowsTerminal/WindowsTerminal.manifest [hunk 1]
```diff
@@ -14,6 +14,11 @@
         <!-- This will cause unpackaged activation failures in XAML Islands. -->
         <!-- (We use unpackaged activation in test scenarios.) See GH#10265. -->
         <maxversiontested Id="10.0.18362.0"/>
+
+        <!-- You apparently are expected to include all the maxversiontested
+        entries seperately. They are treated like an array. This one turns on Segoe
```
Comment (zadjii-msft) [inline src/cascadia/WindowsTerminal/WindowsTerminal.manifest:19]:
```suggestion
        entries separately. They are treated like an array. This one turns on Segoe
```

**Comment by miniksa** [comment]
Sure you don't want to just shotgun every RTM build number into there between 18362 and 22000 while you're at it?

**Comment by DHowett** [comment]
You kept an eye out for the other Quirks, right?

**Review by DHowett — APPROVED** [review]

File: src/cascadia/WindowsTerminal/WindowsTerminal.manifest (hunk 1 above)
Comment (DHowett) [reply src/cascadia/WindowsTerminal/WindowsTerminal.manifest]: every dang time!

**Review by miniksa — APPROVED** [review]
I'm fine with this, but I'm still wondering if you shouldn't just pour all the versions in here.

**Review by carlos-zamora — APPROVED** [review]

**Comment by DHowett** [comment]
@msftbot merge this in 1 minutes

File: src/cascadia/WindowsTerminal/WindowsTerminal.manifest [hunk 2]
```diff
@@ -14,6 +14,11 @@
         <!-- This will cause unpackaged activation failures in XAML Islands. -->
         <!-- (We use unpackaged activation in test scenarios.) See GH#10265. -->
         <maxversiontested Id="10.0.18362.0"/>
+
+        <!-- You apparently are expected to include all the maxversiontested
+        entries separately. They are treated like an array. This one turns on Segoe
+        UI Variable, GH #12452 -->
```
Comment (lhecker) [inline src/cascadia/WindowsTerminal/WindowsTerminal.manifest:20]: Are you all sure this is true? I've looked into the source code behind this field and it doesn't indicate that it supports more than one value here.

I believe this value is stored at
```
HKEY_CLASSES_ROOT\Local Settings\Software\Microsoft\Windows\CurrentVersion\AppModel\Repository\Packages\*\OSMaxVersionTested
```
which only supports a single value. See `AppModelVersionP.hpp`.

Comment (DHowett) [reply src/cascadia/WindowsTerminal/WindowsTerminal.manifest:20]: That may be true of the _appxmanifest_, but it is not true of the fusion manifest :smile:

Comment (DHowett) [reply src/cascadia/WindowsTerminal/WindowsTerminal.manifest:20]: See MSFT-33672605 for more info!

Comment (lhecker) [reply src/cascadia/WindowsTerminal/WindowsTerminal.manifest:20]: Nope. For future reference MSFT:33672605.

**Pull request merged**